### ISSUE TITLE
Вылетает при попытке подобрать предмет (Crashes when trying to pick up an item) MOD: TWHL Tower 2

### ISSUE THREAD

**Issue by DENCHIKTEAM** [FWGS/xash3d-fwgs#723]
При попытке подобрать предмет в моде TWHL Tower 2 в главе Memories игра сворачивается в главное меню
![Screenshot_13](https://user-images.githubusercontent.com/51756144/148437750-53d4f64f-23a4-4f07-8138-334eda3b9de8.png)

**Comment by DENCHIKTEAM** [comment]
Вот как это выглядит на видео
https://youtu.be/UfYmhxrZ_Ho

**Comment by DENCHIKTEAM** [comment]
Играл я на ПК с Windows 10, версия движка тестовая v49/0.20 win32-i386 build 2470

[engine.log](https://github.com/FWGS/xash3d-fwgs/files/7825450/engine.log)

**Comment by a1batross** [comment]
Пофиксилось что ли?

**Comment by DENCHIKTEAM** [comment]
Да, этот вылет походу был исправлен. Буквально пару часов назад я полностью прошел этот мод на 100 (т. е. собрал все диски). Все подбираемые предметы работают в главах Memories, Not Much Of A Turnabout, Trial By Fire и т. д. У меня игра вылетала полностью пару раз на главах Not Much Of A Turnabout и Von Braun, но все это решается повторным запуском игры и загрузкой сохранения. Эти вылеты определенно не связаны с предметами (я надеюсь).

**Comment by a1batross** [comment]
Хорошо было бы понять что за вылеты тогда.

**Comment by DENCHIKTEAM** [comment]
Вот затриггерил в главе Von Braun на электронной панели где надо вводить код, первый вылет на этой главе у меня был когда я подбирал диск для прохождения на 100%
![Screenshot_18](https://user-images.githubusercontent.com/51756144/210180516-dd0912ab-ab9c-44e5-b42d-c6df189e359b.png)
![Screenshot_19](https://user-images.githubusercontent.com/51756144/210180517-d66e0587-6fbc-496f-8ae6-a9db8bf2dd4b.png)

**Comment by DENCHIKTEAM** [comment]
А вот уже тот самый вылет на месте где диск лежит
![Screenshot_20](https://user-images.githubusercontent.com/51756144/210180755-dda2860f-9b22-4a3f-83a9-799c5243a9c0.png)
![Screenshot_21](https://user-images.githubusercontent.com/51756144/210180758-c317c7ad-d951-44f8-b828-c4c11d3de552.png)
И вот лог сессии с параметром -dev 3
[engine.log](https://github.com/FWGS/xash3d-fwgs/files/10329240/engine.log)

**Comment by erfty125** [comment]
> А вот уже тот самый вылет на месте где диск лежит
> ![Screenshot_20](https://user-images.githubusercontent.com/51756144/210180755-dda2860f-9b22-4a3f-83a9-799c5243a9c0.png)
> ![Screenshot_21](https://user-images.githubusercontent.com/51756144/210180758-c317c7ad-d951-44f8-b828-c4c11d3de552.png)
> И вот лог сессии с параметром -dev 3
> [engine.log](https://github.com/FWGS/xash3d-fwgs/files/10329240/engine.log)
> 

возможно проблема в ref_gles(1,2).dll. Точно не знаю но если не находит модуль , это явно не нормально

**Comment by erfty125** [comment]
> А вот уже тот самый вылет на месте где диск лежит
> ![Screenshot_20](https://user-images.githubusercontent.com/51756144/210180755-dda2860f-9b22-4a3f-83a9-799c5243a9c0.png)
> ![Screenshot_21](https://user-images.githubusercontent.com/51756144/210180758-c317c7ad-d951-44f8-b828-c4c11d3de552.png)
> И вот лог сессии с параметром -dev 3
> [engine.log](https://github.com/FWGS/xash3d-fwgs/files/10329240/engine.log)
> 

скинь лог после перезагрузки и загрузки сохранения(когда игра не вылетает)

**Comment by DENCHIKTEAM** [comment]
> > А вот уже тот самый вылет на месте где диск лежит
> > ![Screenshot_20](https://user-images.githubusercontent.com/51756144/210180755-dda2860f-9b22-4a3f-83a9-799c5243a9c0.png)
> > ![Screenshot_21](https://user-images.githubusercontent.com/51756144/210180758-c317c7ad-d951-44f8-b828-c4c11d3de552.png)
> > И вот лог сессии с параметром -dev 3
> > [engine.log](https://github.com/FWGS/xash3d-fwgs/files/10329240/engine.log)
> 
> скинь лог после перезагрузки и загрузки сохранения(когда игра не вылетает)

Вот пожалуйста, запустил мод и загрузил сохранение и допрошёл главу
[engine.log](https://github.com/FWGS/xash3d-fwgs/files/10329330/engine.log)

**Comment by Vladislav4KZ** [comment]
@erfty125 в Windows нет OpenGL ES рендерера, там обычный OpenGL.

И в Windows версии ксаша вообще используется ref_gl.dll

А ref_soft.dll это Software рендерер.
